go.mod

```diff
@@ -51,18 +50,18 @@ require (
 	github.com/ryanuber/go-glob v1.0.0
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/chains v0.11.0
+	github.com/tektoncd/chains v0.12.1-0.20220901150427-1bf8faaf4475
 	github.com/tektoncd/hub v1.8.0
-	github.com/tektoncd/pipeline v0.37.2
+	github.com/tektoncd/pipeline v0.38.3
```

Comment:
because of the deprecation notice

```
Rename ArrayOrString to ParamValues (https://github.com/tektoncd/pipeline/pull/5304)
Rename ArrayOrString to ParamValues, NewArrayOrString to NewStructuredValues

This deprecation notice is applicable to the projects such CLI, Dashboard, Chains, etc which are dependent on the go types defined in the Pipeline.
```

Reply:
And I think a few more changes will also be required wrt status, i have seen few errors

Reply:
👍🏼